lib/doc/jsdoc-template/publish.js

```diff
@@ -36,6 +36,24 @@ var prettifyCode = R.pipe(
   R.prop('value')
 );
 
+// handle any combination of comma-separated and multi-line @see annotations
+var simplifySee = R.pipe(
+  // see is array of 0 or more @see lines, map over them
+  R.map(
+    R.pipe(
+      // each line may or may not be comma separated
+      R.split(','),
+      R.map(
+        R.pipe(
+          R.trim,
+          unprefix
+        )
+      )
+    )
+  ),
+  R.flatten
+);
```

Comment:
input: `['R.one', 'R.two, R.three']`
output: `['one', 'two', 'three']`

also I'm in ramda's gitter, maybe easier if you jump in there?